swagger.yaml

```diff
@@ -79,6 +79,28 @@ paths:
           description: "Unauthorised to fetch"
         500:
           $ref: "#/responses/InternalError"
+  /population-types/{population-type}:
```

Comment:
I can't find this endpoint in https://github.com/ONSdigital/dp-population-types-api/blob/develop/service/routes.go, what am I missing? Is this in preparation for the implementation?

Reply:
Hi Rafa, yes this is in preparation for the work that needs to be done by the team.